app/shell-window/pages.js

```diff
@@ -37,6 +38,7 @@ var events = new EventEmitter()
 var webviewsDiv = document.getElementById('webviews')
 var closedURLs = []
 var cachedMarkdownRendererScript
+var fullscreenWarning = null
```

Comment:
This doesnt need to be a global, does it?

Reply:
right, we could make it an attribute of `page` : `page.fullscreenWarning` ? 

I wanted to store it so that I could run `promptbar.remove` and was having some problems. This is related to me messing with that `indexOf`